pam_authz/pam/README.md

```diff
@@ -0,0 +1,64 @@
+# Styra's PAM authorization module
+
+This implements a PAM module that integrates with Open Policy Agent (OPA)
+to make authorization decisions. When invoked, the module queries OPA
+and if no errors are returned, then the action is allowed. This allows
+administrators to implement fine-grained access control. For
+example, one can use this module to limit usage of `sudo` to particular
+times of day.
```

Comment:
I will remove everything about "why is pam_authz useful" from here. That info will be found in the top-level README (contrib/pam_authz/README). This document will then only cover how pam_authz works